server/model/pipeline.go

```diff
@@ -52,6 +52,7 @@ type Pipeline struct {
 	Files               []*File           `json:"files,omitempty"         xorm:"-"`
 	ChangedFiles        []string          `json:"changed_files,omitempty" xorm:"json 'changed_files'"`
 	AdditionalVariables map[string]string `json:"variables,omitempty"     xorm:"json 'additional_variables'"`
+	Labels              []string          `json:"labels,omitempty"        xorm:"-"`
```

Comment:
I guess we need to save them otherwise we can't restart a pipeline.

Reply:
xorm:"json 'labels'"